editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -3389,7 +3389,7 @@ void SpatialEditorViewport::drop_data_fw(const Point2 &p_point, const Variant &p
 		if (root_node) {
 			list.push_back(root_node);
 		} else {
-			accept->get_ok()->set_text(TTR("OK :("));
+			accept->get_ok()->set_text(TTR("OK"));
```

Comment:
`RTR("OK")` is already the default value for the `AcceptDialog::ok` button, so all those overrides are unnecessary (unless the value gets overridden somewhere else to a different text).

Reply:
Merging nevertheless as it doesn't hurt as is, but most/all of those should likely be redundant and could be removed in another PR.